src/tvhpoll.c

```diff
@@ -70,7 +70,7 @@ tvhpoll_create ( size_t n )
 {
   int fd;
 #if ENABLE_EPOLL
-  if ((fd = epoll_create1(EPOLL_CLOEXEC)) < 0) {
+  if ((fd = epoll_create(1)) < 0) {
```

Comment:
Agreed again, but same issue as with inotify. It does not seem to be selected for PPC architectures. Please see the inotify case above.

On another note: I was wondering whether configuring with `--disable-epoll` would circumvent the problem like with `--disable-inotify` but the epoll option does not seem to make any difference. This is in line with the code in `configure`, where the only condition for epoll is `linux` architecture but no reference to the configuration option is made. However, fixing that led to even more issues, which is why I gave up this route...